[BUG] Resetting password via mail leads to internal server error

### Description

Due to:

https://github.com/paperless-ngx/paperless-ngx/blob/6d25eb26a193d5372ca31ad7920960daf5667137/src/documents/templates/account/password_reset_from_key_done.html#L12-L15

using `endblock` to start a block, after a password reset is done via email, an internal server error gets displayed.

I have a trivial fix here: https://github.com/paperless-ngx/paperless-ngx/commit/68be4bd0fcfeaecdcda5bb8378c69004382db76e

But I'm currently running out of time to take care of adding a test for this ideally. If someone wants to pick it up, I'd be glad, otherwise I'll be able to pick this up again in early May.

### Steps to reproduce

- Use "forgot password" in the log-in screen
- Set a new password via email link

### Webserver logs

```pytb
webserver-1  | [2024-04-19 17:41:33,305] [ERROR] [django.request] Internal Server Error: /accounts/password/reset/key/done/
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 505, in parse
webserver-1  |     compile_func = self.tags[command]
webserver-1  |                    ~~~~~~~~~^^^^^^^^^
webserver-1  | KeyError: 'endblock'
webserver-1  | 
webserver-1  | During handling of the above exception, another exception occurred:
webserver-1  | 
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 42, in inner
webserver-1  |     response = await get_response(request)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 284, in _get_response_async
webserver-1  |     response = await sync_to_async(
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 468, in __call__
webserver-1  |     ret = await asyncio.shield(exec_coro)
webserver-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/current_thread_executor.py", line 40, in run
webserver-1  |     result = self.fn(*self.args, **self.kwargs)
webserver-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 522, in thread_handler
webserver-1  |     return func(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 114, in render
webserver-1  |     self.content = self.rendered_content
webserver-1  |                    ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 90, in rendered_content
webserver-1  |     template = self.resolve_template(self.template_name)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 72, in resolve_template
webserver-1  |     return select_template(template, using=self.using)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loader.py", line 42, in select_template
webserver-1  |     return engine.get_template(template_name)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/backends/django.py", line 33, in get_template
webserver-1  |     return Template(self.engine.get_template(template_name), self)
webserver-1  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/engine.py", line 175, in get_template
webserver-1  |     template, origin = self.find_template(template_name)
webserver-1  |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/engine.py", line 157, in find_template
webserver-1  |     template = loader.get_template(name, skip=skip)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
webserver-1  |     template = super().get_template(template_name, skip)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
webserver-1  |     return Template(
webserver-1  |            ^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
webserver-1  |     self.nodelist = self.compile_nodelist()
webserver-1  |                     ^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 200, in compile_nodelist
webserver-1  |     return parser.parse()
webserver-1  |            ^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 513, in parse
webserver-1  |     raise self.error(token, e)
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 511, in parse
webserver-1  |     compiled_result = compile_func(self, token)
webserver-1  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loader_tags.py", line 293, in do_extends
webserver-1  |     nodelist = parser.parse()
webserver-1  |                ^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 507, in parse
webserver-1  |     self.invalid_block_tag(token, command, parse_until)
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 568, in invalid_block_tag
webserver-1  |     raise self.error(
webserver-1  | django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 12: 'endblock'. Did you forget to register or load this tag?
webserver-1  | [2024-04-19 17:41:39,973] [ERROR] [django.request] Internal Server Error: /accounts/password/reset/key/done/
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 505, in parse
webserver-1  |     compile_func = self.tags[command]
webserver-1  |                    ~~~~~~~~~^^^^^^^^^
webserver-1  | KeyError: 'endblock'
webserver-1  | 
webserver-1  | During handling of the above exception, another exception occurred:
webserver-1  | 
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 42, in inner
webserver-1  |     response = await get_response(request)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 284, in _get_response_async
webserver-1  |     response = await sync_to_async(
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 468, in __call__
webserver-1  |     ret = await asyncio.shield(exec_coro)
webserver-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/current_thread_executor.py", line 40, in run
webserver-1  |     result = self.fn(*self.args, **self.kwargs)
webserver-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 522, in thread_handler
webserver-1  |     return func(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 114, in render
webserver-1  |     self.content = self.rendered_content
webserver-1  |                    ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 90, in rendered_content
webserver-1  |     template = self.resolve_template(self.template_name)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/response.py", line 72, in resolve_template
webserver-1  |     return select_template(template, using=self.using)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loader.py", line 42, in select_template
webserver-1  |     return engine.get_template(template_name)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/backends/django.py", line 33, in get_template
webserver-1  |     return Template(self.engine.get_template(template_name), self)
webserver-1  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/engine.py", line 175, in get_template
webserver-1  |     template, origin = self.find_template(template_name)
webserver-1  |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/engine.py", line 157, in find_template
webserver-1  |     template = loader.get_template(name, skip=skip)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
webserver-1  |     template = super().get_template(template_name, skip)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
webserver-1  |     return Template(
webserver-1  |            ^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
webserver-1  |     self.nodelist = self.compile_nodelist()
webserver-1  |                     ^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 200, in compile_nodelist
webserver-1  |     return parser.parse()
webserver-1  |            ^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 513, in parse
webserver-1  |     raise self.error(token, e)
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 511, in parse
webserver-1  |     compiled_result = compile_func(self, token)
webserver-1  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/loader_tags.py", line 293, in do_extends
webserver-1  |     nodelist = parser.parse()
webserver-1  |                ^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 507, in parse
webserver-1  |     self.invalid_block_tag(token, command, parse_until)
webserver-1  |   File "/usr/local/lib/python3.11/site-packages/django/template/base.py", line 568, in invalid_block_tag
webserver-1  |     raise self.error(
webserver-1  | django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 12: 'endblock'. Did you forget to register or load this tag?
```


### Browser logs

```bash
Server Error (500)
```


### Paperless-ngx version

2.7.2

### Host OS

Archlinux

### Installation method

Docker - official image

### Browser

_No response_

### Configuration changes

_No response_

### Other

_No response_

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

--- a/src/documents/templates/account/password_reset_from_key_done.html
+++ b/src/documents/templates/account/password_reset_from_key_done.html
@@ -9,7 +9,7 @@
     <h4>{% translate "Password reset complete." %}</h4>
 {% endblock form_top_content %}
 
-{% endblock form_content %}
+{% block form_content %}
     {% url 'account_login' as login_url %}
     <p>{% blocktranslate %}Your new password has been set. You can now <a href="{{ login_url }}">log in</a>{% endblocktranslate %}.</p>
 {% endblock form_content %}

